ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.10.0__variant.yaml

```diff
@@ -0,0 +1,127 @@
+tag_specification:
+  name: '4.2'
```

Comment:
if we skip 4.1, let's name this file `openshift-knative-eventing-contrib-release-v0.10.0.yaml`